### ISSUE TITLE
Ventcrawling bat bug

### ISSUE THREAD

**Issue by SinfulBliss** [tgstation/tgstation#62430]
Round ID: [172108](https://scrubby.melonmesa.com/round/172108)

Reproduction: https://streamable.com/td2pxg

Switching to bat and ventcrawling shows all the pipes and lets you travel through them. If you go human in a pipe, you can still ventcrawl but you take pressure and temperature damage. If you switch back to bat in the vents as a human, you stay without the ventcrawl as if you were a human, although you don't take the damage.

tl;dr: There's no way to get out of vents if you switch to human in them.

**Comment by TheBonded** [comment]
duplicate of https://github.com/tgstation/tgstation/issues/62424
